src/main/java/org/jenkinsci/test/acceptance/controller/JenkinsController.java

```diff
@@ -164,6 +165,11 @@ public boolean isRunning(){
     @Override
     public abstract URL getUrl();
 
+    @CheckForNull
+    public Credentials getCredentials() {
```

Comment:
Should this be named 'initialCredentials'. They will become useless once a securotyRealm is configured won't they?

Reply:
Renamed.